pkg/reconciler/gcs/gcs.go

```diff
@@ -306,10 +315,11 @@ func (c *Reconciler) reconcileNotification(gcs *v1alpha1.GCSSource) (*storage.No
 
 	c.Logger.Infof("Creating a notification on bucket %s", gcs.Spec.Bucket)
 	notification, err := bucket.AddNotification(ctx, &storage.Notification{
-		TopicProjectID:   gcs.Spec.GoogleCloudProject,
-		TopicID:          gcs.Status.Topic,
-		PayloadFormat:    storage.JSONPayload,
-		EventTypes:       gcs.Spec.EventTypes,
+		TopicProjectID: gcs.Spec.GoogleCloudProject,
+		TopicID:        gcs.Status.Topic,
+		PayloadFormat:  storage.JSONPayload,
+		// Translating the eventType. Allowing only one for the demo.
+		EventTypes:       []string{gcsEventTypes[gcs.Spec.EventType]},
```

Comment:
This is actually a comment on https://github.com/nachocano/gcs/pull/1/files#diff-71518e9d7cfa8b246a7d8fd2d50347abR314:

```go
customAttributes["ce-type"] = "google.gcs"
```

We can change this to 

```go
customAttributes["ce-type"] = gcs.Spec.EventType
```

Reply:
Done, thanks!!!